static/src/js/containers/SpatialDisplayContainer/__tests__/SpatialDisplayContainer.test.js

```diff
@@ -107,11 +107,12 @@ describe('SpatialDisplayContainer component', () => {
       drawingNewLayer: false,
       lineSearch: ['Test Line'],
       onChangeQuery: props.onChangeQuery,
-      onRemoveGridFilter: props.onRemoveGridFilter,
       onRemoveSpatialFilter: props.onRemoveSpatialFilter,
       pointSearch: ['Test Point Search'],
       polygonSearch: ['Test Polygon Search'],
-      shapefile: {}
+      shapefile: {},
+      defaultError: '',
```

Comment:
Lets see if these components are needed we may be able to remove them and just set the initial states to empty strings